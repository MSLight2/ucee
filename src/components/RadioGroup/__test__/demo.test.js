import { demoTest } from '../../../tests/shared/demoTest';
import RadioGroupDemo from '../demo';

demoTest('RadioGroup', RadioGroupDemo);