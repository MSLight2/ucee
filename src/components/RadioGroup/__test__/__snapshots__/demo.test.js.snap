// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioGroup demo test 1`] = `
Array [
  <div>
    option配置
  </div>,
  <div
    className="ucee-radiogroup"
  >
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup1"
            onChange={[Function]}
            type="radio"
            value={1}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option1
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup1"
            onChange={[Function]}
            type="radio"
            value={2}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option2
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup1"
            onChange={[Function]}
            type="radio"
            value={3}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option3
        </span>
      </label>
    </div>
  </div>,
  <div
    className="ucee-radiogroup"
  >
    <div
      className="ucee-radio--disabled"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            disabled={true}
            name="radioGroup"
            onChange={[Function]}
            type="radio"
            value={1}
          />
          <span
            className="ucee-radio__inner"
            style={
              Object {
                "color": "#c8c9cc",
              }
            }
          >
            <i
              className="ucee-radio__inner-active"
              style={
                Object {
                  "background": "#c8c9cc",
                }
              }
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option1
        </span>
      </label>
    </div>
    <div
      className="ucee-radio--disabled"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            disabled={true}
            name="radioGroup"
            onChange={[Function]}
            type="radio"
            value={2}
          />
          <span
            className="ucee-radio__inner"
            style={
              Object {
                "color": "#c8c9cc",
              }
            }
          >
            <i
              className="ucee-radio__inner-active"
              style={
                Object {
                  "background": "#c8c9cc",
                }
              }
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option2
        </span>
      </label>
    </div>
    <div
      className="ucee-radio--disabled"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            disabled={true}
            name="radioGroup"
            onChange={[Function]}
            type="radio"
            value={3}
          />
          <span
            className="ucee-radio__inner"
            style={
              Object {
                "color": "#c8c9cc",
              }
            }
          >
            <i
              className="ucee-radio__inner-active"
              style={
                Object {
                  "background": "#c8c9cc",
                }
              }
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option3
        </span>
      </label>
    </div>
  </div>,
  <div
    className="ucee-radiogroup"
  >
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup3"
            onChange={[Function]}
            type="radio"
            value={1}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option1
        </span>
      </label>
    </div>
    <div
      className="ucee-radio--checked"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={true}
            name="radioGroup3"
            onChange={[Function]}
            type="radio"
            value={2}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option2
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup3"
            onChange={[Function]}
            type="radio"
            value={3}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option3
        </span>
      </label>
    </div>
  </div>,
  <div>
    水平展示
  </div>,
  <div
    className="ucee-radiogroup ucee-radiogroup--horizontal"
  >
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup-horizontal"
            onChange={[Function]}
            type="radio"
            value={1}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option1
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup-horizontal"
            onChange={[Function]}
            type="radio"
            value={2}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option2
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="radioGroup-horizontal"
            onChange={[Function]}
            type="radio"
            value={3}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          option3
        </span>
      </label>
    </div>
  </div>,
  <div>
    组合Radio
  </div>,
  <div
    className="ucee-radiogroup"
  >
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="renderGroup1"
            onChange={[Function]}
            type="radio"
            value={1}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          默认
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="renderGroup1"
            onChange={[Function]}
            type="radio"
            value={2}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          默认
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="renderGroup1"
            onChange={[Function]}
            type="radio"
            value={3}
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          默认
        </span>
      </label>
    </div>
  </div>,
  <div>
    disabled
  </div>,
  <div
    className="ucee-radiogroup"
  >
    <div
      className="ucee-radio--disabled"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            disabled={true}
            name="renderGroup2"
            onChange={[Function]}
            type="radio"
            value={1}
          />
          <span
            className="ucee-radio__inner"
            style={
              Object {
                "color": "#c8c9cc",
              }
            }
          >
            <i
              className="ucee-radio__inner-active"
              style={
                Object {
                  "background": "#c8c9cc",
                }
              }
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          禁用
        </span>
      </label>
    </div>
    <div
      className="ucee-radio--disabled ucee-radio--checked"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={true}
            disabled={true}
            name="renderGroup2"
            onChange={[Function]}
            type="radio"
            value={2}
          />
          <span
            className="ucee-radio__inner"
            style={
              Object {
                "color": "#c8c9cc",
              }
            }
          >
            <i
              className="ucee-radio__inner-active"
              style={
                Object {
                  "background": "#c8c9cc",
                }
              }
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          禁用
        </span>
      </label>
    </div>
    <div
      className="ucee-radio--disabled"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            disabled={true}
            name="renderGroup2"
            onChange={[Function]}
            type="radio"
            value={3}
          />
          <span
            className="ucee-radio__inner"
            style={
              Object {
                "color": "#c8c9cc",
              }
            }
          >
            <i
              className="ucee-radio__inner-active"
              style={
                Object {
                  "background": "#c8c9cc",
                }
              }
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          禁用
        </span>
      </label>
    </div>
  </div>,
  <div>
    onChange
  </div>,
  <div
    className="ucee-radiogroup"
  >
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="renderGroup3"
            onChange={[Function]}
            type="radio"
            value="change1"
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          change1
        </span>
      </label>
    </div>
    <div
      className="ucee-radio--checked"
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={true}
            name="renderGroup3"
            onChange={[Function]}
            type="radio"
            value="change2"
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          change2
        </span>
      </label>
    </div>
    <div
      className=""
    >
      <label
        className="ucee-radio__wrapper"
      >
        <span
          className="ucee-radio"
        >
          <input
            className="ucee-radio__input"
            defaultChecked={false}
            name="renderGroup3"
            onChange={[Function]}
            type="radio"
            value="change3"
          />
          <span
            className="ucee-radio__inner"
          >
            <i
              className="ucee-radio__inner-active"
            />
          </span>
        </span>
        <span
          className="ucee-radio__text"
        >
          change3
        </span>
      </label>
    </div>
  </div>,
]
`;
